File: src/main/java/com/otaviolarrosa/identitymanager/domain/utils/StringUtils.java
package com.otaviolarrosa.identitymanager.domain.utils;

public class StringUtils {
	public static boolean StringIsNullOrEmptyOrWhitespace(String str) {
		return (str.equals(null) || str.equals("") || str.equals(" "));
	}
	
	public static boolean HasMinimumLenght(String str, int minimunLenght) {
		return str.length() >= minimunLenght;
	}
}

File: src/main/kotlin/com/otaviolarrosa/singleSignOn/application/commons/utils/StringUtils.kt
package com.otaviolarrosa.singleSignOn.application.commons.utils

object StringUtils {
    const val STRING_EMPTY = ""


    fun stringIsNullOrEmptyOrWhitespace(str: String?): Boolean {
        return (str == null || str.trim() == "" || str.trim() == " ")
    }

    fun hasMinimumLenght(str: String, minimunLenght: Int): Boolean {
        return str.length >= minimunLenght
    }

    fun hasMaximumLenght(str: String, minimunLenght: Int): Boolean {
        return str.length <= minimunLenght
    }
}